src/components/ChangePlan/CheckoutProcess.js

```diff
@@ -0,0 +1,361 @@
+import React, { useEffect, useState } from 'react';
+import { InjectAppServices } from '../../services/pure-di';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+import { useIntl } from 'react-intl';
+import { useRouteMatch } from 'react-router-dom';
+import { Loading } from '../Loading/Loading';
+import {
+  FieldGroup,
+  FieldItem,
+  FormWithCaptcha,
+  InputFieldItem,
+  PhoneFieldItem,
+  SelectFieldItem,
+  SubmitButton,
+} from '../form-helpers/form-helpers';
+import { Helmet } from 'react-helmet';
+
+const ContactInformation = ({
+  contactInformation,
+  countries,
+  steps,
+  setActiveStep,
+  handleSaveAndContinue,
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const fieldNames = {
+    firstname: 'firstname',
+    lastname: 'lastname',
+    address: 'address',
+    city: 'city',
+    province: 'province',
+    country: 'country',
+    zipcode: 'zipcode',
+    phone: 'phone',
+    company: 'company',
+    industry: 'industry',
+  };
+
+  const getFormInitialValues = () => {
+    const initialValues = Object.keys(fieldNames).reduce(
+      (accumulator, currentValue) => ({
+        ...accumulator,
+        [currentValue]: '',
+      }),
+      {},
+    );
+
+    if (contactInformation !== null) {
+      initialValues[fieldNames.firstname] = contactInformation.firstname;
+      initialValues[fieldNames.lastname] = contactInformation.lastname;
+      initialValues[fieldNames.address] = contactInformation.address;
+      initialValues[fieldNames.city] = contactInformation.city;
+      initialValues[fieldNames.province] = contactInformation.province;
+      initialValues[fieldNames.country] = contactInformation.countryId;
+      initialValues[fieldNames.zipcode] = contactInformation.zipCode;
+      initialValues[fieldNames.phone] = contactInformation.phoneNumber;
+      initialValues[fieldNames.company] = contactInformation.company;
+      initialValues[fieldNames.industry] = contactInformation.industry;
+    }
+
+    return initialValues;
+  };
+
+  const [formSubmitted, setFormSubmitted] = useState(false);
+  const [isCompletedStep, setCompletedStep] = useState(false);
+
+  const submitContactInformationForm = async (values, { setSubmitting }) => {
+    setFormSubmitted(false);
+    try {
+      const result = await handleSaveAndContinue(values);
+      if (result.success) {
+        setFormSubmitted(true);
+        setCompletedStep(true);
+      }
+    } finally {
+      setTimeout(() => setFormSubmitted(false), 3000);
```

Comment:
You could use **`const createTimeout = useTimeout();`**
```suggestion
createTimeout(() => setFormSubmitted(false), 3000);
```